data_node/upload_manager.go

```diff
@@ -67,7 +71,7 @@ func (um *UploadManager) handleUpload(w http.ResponseWriter, r *http.Request, _
 func (um *UploadManager) handleInitialUpload(w http.ResponseWriter, r *http.Request) {
 	log.Println(um.logPrefix, r.RemoteAddr, "Received INIT request")
 
-	expectedHeaders := []string{"Content-Length", "Filename"}
+	expectedHeaders := []string{"Filename", "Filesize"}
```

Comment:
Can we use the same convention for header names? Some of the headers are named like `Headername` while some are named `Header-Name`. I think let's use the later in all headers.

Reply:
for some reason, any property related to file are done without space (filename, filesize ,,, etc). and the usual pattern for headers are hyphened (content-lengeth, keep-alive ,,, etc)